policy/modules/services/tpm2.fc

```diff
@@ -1,4 +1,5 @@
 /usr/bin/tpm2-abrmd								--	gen_context(system_u:object_r:tpm2_abrmd_exec_t,s0)
+/usr/bin/tpm2_.*									--	gen_context(system_u:object_r:tpm2_exec_t,s0)
```

Comment:
Need to replace `.*` with `.+`. This showed up in the CI build, though I'm not sure why the result isn't blocking the PR.

Reply:
I think I fixed the CI integration.